h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
We must, because they don't give us the information that we desperately need.
This is an 4 year old library, nobody works on it any more. I had to fork it.